libr/core/cmd_search.c

```diff
@@ -938,6 +937,13 @@ static void print_rop (RCore *core, RList *hitlist, char mode, bool *json_first)
 			}
 			free (buf);
 		}
+		if (esil && hit) {
+			r_cons_printf ("Gadget size: %d\n", size);
+			char input[80] = { 0 };
```

Comment:
define variables at the begining of function frames. and no need to initialize it